node/src/main/scala/com/wavesplatform/ImporterSending.scala

```diff
@@ -0,0 +1,82 @@
+package com.wavesplatform
+import java.io.BufferedInputStream
+
+import akka.actor.ActorSystem
+import com.wavesplatform.Importer.{initBlockchain, initFileStream, initTime, initUtxPool, loadSettings, parseOptions, startImport}
+import com.wavesplatform.account.{Address, AddressScheme}
+import com.wavesplatform.extensions.{Context, Extension}
+import com.wavesplatform.settings.WavesSettings
+import com.wavesplatform.state.{Blockchain, BlockchainUpdated}
+import com.wavesplatform.transaction.{Asset, BlockchainUpdater, Transaction}
+import com.wavesplatform.utils.{ScorexLogging, Time}
+import com.wavesplatform.utx.UtxPool
+import com.wavesplatform.wallet.Wallet
+import monix.execution.Scheduler
+import monix.reactive.Observable
+import monix.reactive.subjects.ConcurrentSubject
+
+import scala.concurrent.duration._
+import scala.concurrent.Await
+import scala.util.{Failure, Success, Try}
+
+object ImporterSending extends ScorexLogging {
```

Comment:
The fact that there are two importers, one which publishes updates and another which isn't, hints at some design issues. Importer should not care about node configuration, it just feeds blocks to the node, no matter which plugins are enabled.